app/views/admin/stolen_bikes/edit.html.haml

```diff
@@ -1,110 +1,135 @@
-      #alert-block
-        .alert.alert-error
-          %a.close{"data-dismiss" => "alert"} ×
-          %h4
-            Please fix the following 
-            = pluralize(@customer_contact.errors.count, "error")
-          %ul
-            - @customer_contact.errors.full_messages.each do |msg|
-              %li 
-                = msg  
-    .feedbacks
-      %h4
-        Send bike owner an email
-      .contact-info
-        .contact-first-row
-          = f.email_field :creator_email, value: current_user.email
-          = f.email_field :user_email, value: @bike.owner_email
-        = f.text_field :title, value: "Your stolen #{@bike.title_string}"
-        = f.hidden_field :contact_type, value: 'stolen_contact'
-        = f.hidden_field :creator_id, value: current_user.id
-        = f.hidden_field :bike_id, value: @bike.id
-      .contact-text
-        = f.text_area :body, placeholder: "You can write HTML in here. There will be a link to the #{@bike.type} included automatically.", rows: 7   
-      .feedback-actions.control-group.padded
-        = f.submit 'Send it!', class: 'button-green pull-right'
-    
+      = render partial: "/shared/errors", locals: { name: "Customer contact", obj: @customer_contact}
 
-  #chosen-container
-    %ul.bike-boxes
-      = render partial: "/bikes/bike", locals: {bike: @bike, target: "show" }
-    - if @stolen_record.present?
-      %p
-        = @stolen_record.theft_description
-      .dl-holder
-        %dl.dl-horizontal
-          = @bike.dl_list_item(@bike.created_at.strftime("%m-%d-%Y"), "Created")
-          = @bike.dl_from_attribute("owner_email")
-          = @bike.dl_list_item(@bike.owner && @bike.owner.email, "Current Owner")
-          - if @bike.creation_organization_id.present?
-            = @bike.dl_list_item(@bike.creation_organization.name, "Creation Organization")
-          %dt
+    %h4
+      Send bike owner an email
+    .form-group
+      = f.label :current_user
+      = f.email_field :creator_email, value: current_user.email, class: "form-control w-25"
+    .form-group
+      = f.label :user_email
+      = f.email_field :user_email, value: @bike.owner_email, class: "form-control w-25"
+    .form-group
+      = f.label :bike, "Your stolen bike"
+      = f.text_field :title, value: "Your stolen #{@bike.title_string}", class: "form-control w-25"
+    = f.hidden_field :contact_type, value: 'stolen_contact'
+    = f.hidden_field :creator_id, value: current_user.id
+    = f.hidden_field :bike_id, value: @bike.id
+    .form-group
+      = f.label :notification_body
+      = f.text_area :body, placeholder: "You can write HTML in here. There will be a link to the #{@bike.type} included automatically.", rows: 7, class: "form-control w-25"
+    .row
+      .col-sm
+      .col-sm
+      .col-sm
+        = f.submit 'Send it!', class: 'btn btn-success'
+
+  %ul.bike-boxes
+    -# Should I create a new partial here?
+    = render partial: "/bikes/bike", locals: {bike: @bike, target: "show" }
```

Comment:
I believe the only place that is using the partial `/bikes/bike` is admin. Look around and make sure?

If that is the case, you should totally create a new partial - put it in `admin/bikes/bike` bonus points if you delete `/bikes/bike` and replace all the instances in admin with the new admin partial (even if the other places in admin don't look perfect because they aren't using the revised view)